### DISCUSSION TITLE
Как выбрать аудиодорожку с другой озвучкой

### DISCUSSION THREAD

**Discussion by Anatoly4444** [quarckster/kodi.kino.pub#219] [Q&A] [answered]
Использую kodi 18 и не могу найти данной опции. Есть ли такая возможность в 3ей версии аддона?

**Comment by quarckster** [comment, marked as answer]
> Использую kodi 18 и не могу найти данной опции. Есть ли такая возможность в 3ей версии аддона?

Во-первых, третья версия аддона не работает в Kodi 18, используйте 2.12.4. Во-вторых, выбор дорожки возможен с типами медиапотока http и hls4.

**Comment by Avinnare86** [comment]
Включаю hls4 - ошибка при вклюении видео, включаю hls2 \ hls видео показывается с кинопаба, но нет выбора аудиодорожки. Что не так?

**Reply by quarckster** [reply]
> ошибка при вклюении видео

Скорее всего у вам кинопаб заблокирован. Выбор аудиодорожки возможно только для hls4.
